src/components/analyses/listing/Actions.js

```diff
@@ -109,13 +111,26 @@ export default function Actions(props) {
     const isTerminatedAnalysis = isTerminated(analysis);
     return (
         <>
-            <Link href={outputFolderHref} as={outputFolderAs} passHref>
+            {!isVICE && (<Link href={outputFolderHref} as={outputFolderAs} passHref>
                 <GotoOutputFolderButton
                     baseId={baseId}
                     isTerminated={isTerminatedAnalysis}
                     setPendingTerminationDlgOpen={setPendingTerminationDlgOpen}
                 />
-            </Link>
+            </Link>)}
+            {isVICE && !isTerminatedAnalysis && (
+                <IconButton
+                    size="small"
+                    onClick={(event) => {
+                        setVICELogsDlgOpen(true);
+                    }}
+                    id={buildID(baseId, ids.ICONS.LOGS, ids.BUTTON)}
+                    color="primary"
+                    title={t("viewLogs")}
+                >
+                    <LogsIcon fontSize="small" />
+                </IconButton>
+            )}
```

Comment:
For VICE analysis, I removed `Go To output folder` action and add `View Logs` action.